src/_includes/components/logo.njk

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><svg id="Layer_2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 562.9 237.82"><g id="Layer_1-2"><path d="M183.56,237.26h-58.52c12.52-42.59,8.78-75.66,8.74-93.84-.08-38.81-40.5-67.07-81.38-71.04-7.59,34.57-.58,131.52,12.24,164.56H0C17.43,158.84,16.58,80.31,.41,.82H155.44c5.21,16.71,10.84,39.19,10.84,59.34-4.74-6.44-18.15-16.09-22.07-19.6-12.12-10.83-23.63-23.05-37.51-31.04-15.5-8.92-44.27-3.88-52.21,8.28-8.59,13.15-6.59,32.87,5.96,43.97,20.05,17.73,57.08,18.47,81.71,13.07,9.11-2,17.82-5.83,26.88-8.88,7.81,57.29-10.12,115.84,14.53,171.3Z"/><path d="M154.31,236.87c19.71-54.11,19.67-180.76,.21-236.87,2.14,0,2.95,0,4.95,.03,35.59,.4,72.57-.35,108.13,.86,9.73,.33,19.83,3.01,28.92,6.68,25.05,10.12,30.49,37.68,11.52,57.01-6.3,6.42-13.91,11.55-19.49,16.1,13.28,9.05,29.19,16.16,39.96,27.94,34.17,37.36,20.2,94.57-27.99,116.33-13.92,6.29-29.76,10.64-44.94,11.41-33.32,1.69-66.78,.52-101.27,.52Zm150.43-91.39c.32-10.3-3.9-28.21-14.21-44.41-9.9-15.56-11.05-15.79-25.89-5.47-7.11,4.94-14.41,9.86-20.55,15.88-28.14,27.62-42.01,59.87-31.52,99.86,2.61,9.95,8.77,15.26,18.15,18.23,18.88,5.97,43.01-1.53,56.11-18.06,13.28-16.76,17.29-36.47,17.9-66.04Zm-123.63,44.53c1.02,.54,2.04,1.09,3.05,1.63,14.28-42.04,45.77-71.65,72.67-104.73,8.97-11.03,15.22-25.29,19.25-39.08,6.08-20.81-5.62-37.72-25.88-41.75-27.27-5.42-41.11,2.19-45.6,26.29-3.63,19.54-6,39.31-8.92,58.98-4.87,32.88-9.72,65.77-14.59,98.65Z"/><path d="M431.85,127.7c-6.58,34.99,12.86,65.23,45.93,74.52,37.67,10.58,69.44-6.78,84.66-46.24,1.03,7.04,.28,14.58-1.95,22.12-1.66,5.65-4.15,11.3-7.32,16.74-8.84,15.16-23.01,28.67-39.49,35.92-35.13,15.45-78.73,4.87-102.57-24.89-24.17-30.16-25.31-76.9-2.63-107.93,23.12-31.63,66.03-44.17,102.17-29.85,24.41,9.67,38.78,32.36,36.69,59.62h-115.49Zm-.66-4.92c16.02,0,30.67,.44,45.28-.21,6.32-.28,12.82-2.13,18.76-4.46,12.38-4.86,18.16-14.77,16.83-27.47-1.39-13.32-10.73-19.31-22.3-21.76-13.93-2.95-26.5,.98-37.04,10.71-12.23,11.28-18.43,25.57-21.55,43.19Z"/><path d="M383.01,236.47h-50.41c0-19.56-.07-38.46,.02-57.37,.07-15.16,1.81-30.48,.1-45.42-1.71-14.95-6.78-29.53-11.07-47.09,12.75-6.09,29.32-13.99,48.32-23.05v47.6c.97,.44,1.95,.89,2.92,1.33,11.63-30,61.2-46.82,94.92-49.49-15.11,7.36-32.23,25.1-35.37,36-.63,2.18-5.77,4.22-8.9,4.34-36.98,1.41-52.89,17.38-52.78,53.98,.04,13.85-.53,27.89,1.55,41.49,1.97,12.87,7.05,25.26,10.71,37.67Z" fill="#EE4B2B"/></g></svg>
```

Comment:
**TIP!** 
You can optmize the SVG by removing the `<?xml version...` (remember mamboleoo's talk about SVG at fronteers?) and to improve its accessibility try adding a title, description, an id, aria-labelled-by and a role.

Example:
```
<svg id="cat" viewBox="0 0 720 800" aria-labelledby="catTitle catDesc" role="img">
  <title id="catTitle">Pixels, My Super-friendly Cat</title>
  <desc id="catDesc">An illustrated gray cat with bright green blinking eyes.</desc>
```

More info can be found in this article: 
[Accessible SVG's](https://css-tricks.com/accessible-svgs/#aa-2-inline-svg)